cpp/src/neighbors/detail/cagra/search_single_cta_kernel-ext.cuh

```diff
@@ -23,7 +23,7 @@
 namespace cuvs::neighbors::cagra::detail {
 namespace single_cta_search {
 
-#ifdef CUVS_EXPLICIT_INSTANTIATE_ONLY
+#ifdef _CUVS_EXPLICIT_INSTANTIATE_ONLY
```

Comment:
Hi @cjnolet , I'm not so sure if our Cagra module needs to open this macro, so add a `_` for the placeholder. If you feel it is necessary, could you help tag a suitable reviewer to take a look? Thanks!